ui/raidboss/emulator/data/CombatantJobSearch.ts

```diff
@@ -1,14 +1,18 @@
+import { Job } from '../../../../types/job';
+
 export default class CombatantJobSearch {
-  static getJob(abilityId: number): string | undefined {
-    for (const job in CombatantJobSearch.abilities) {
-      if (CombatantJobSearch.abilities[job]?.includes(abilityId))
-        return job;
+  static getJob(abilityId: number): Job | undefined {
+    for (const [key, value] of Object.entries(CombatantJobSearch.abilities)) {
+      if (value) {
+        if (value.includes(abilityId))
+          return key as Job;
+      }
```

Comment:
Can this just be shortened to:
```suggestion
      if (value?.includes(abilityId))
        return key as Job;
```
?